src/satchip/chip_sentinel1rtc.py

```diff
@@ -136,41 +141,38 @@ def _is_valid_rtc_job(job: hyp3_sdk.Job) -> bool:
     return (
         not job.failed()
         and not job.expired()
-        and job.job_parameters['radiometry'] == 'gamma0'
-        and job.job_parameters['resolution'] == 20
+        and job.job_parameters['radiometry'] == 'gamma0'  # type: ignore
+        and job.job_parameters['resolution'] == 20  # type: ignore
     )
 
 
-def _download_hyp3_rtc(job: hyp3_sdk.Job, scratch_dir: Path) -> tuple[Path, Path]:
-    output_path = scratch_dir / job.to_dict()['files'][0]['filename']
+def _download_hyp3_rtc(job: hyp3_sdk.Job, image_dir: Path) -> utils.RtcImageSet:
+    output_path = image_dir / job.to_dict()['files'][0]['filename']
     output_dir = output_path.with_suffix('')
     output_zip = output_path.with_suffix('.zip')
     if not output_dir.exists():
-        job.download_files(location=scratch_dir)
+        job.download_files(location=image_dir)
         hyp3_sdk.util.extract_zipped_product(output_zip)
     vv_path = list(output_dir.glob('*_VV.tif'))[0]
     vh_path = list(output_dir.glob('*_VH.tif'))[0]
-    return vv_path, vh_path
+    image_set = utils.RtcImageSet(vv=vv_path, vh=vh_path)
+    return image_set
 
 
-def get_s1rtc_chip_data(
-    chip: TerraMindChip, image_sets: list[Path], scratch_dir: Path, opts: utils.ChipDataOpts
-) -> xr.DataArray:
+def get_s1rtc_chip_data(chip: TerraMindChip, image_sets: list[utils.RtcImageSet]) -> xr.Dataset:
     roi = shapely.box(*chip.bounds)
-    das = []
     template = create_template_da(chip)
+    timestep_arrays = []
     for image_set in image_sets:
-        for band_name, image_path in zip(['VV', 'VH'], image_set):
+        band_arrays = []
+        for band in S1RTC_BANDS:
+            image_path = image_set._asdict()[band]
```

Comment:
Looking at this why don't we just make it a `TypedDict` with `VV` and `VH` as keys to make it dead simple.